components/bl-badge-component/README.md

```diff
@@ -1,18 +1,44 @@
 # Badge
 
-This is a component of Backendless [UI-Builder](https://backendless.com/developers/#ui-builder) designer. It generates a small badge to the corner of its child(ren). Any text, custom image or Material icon can be as a content of the component. It is also possible to combine them.
+The Badge component allows displaying specific data in the corner of the selected image/icon. The data must be a number or a string value which is fetched from the database and projected up into the right corner. This component is available in the [UI-Builder](https://backendless.com/developers/#ui-builder).
```

Comment:
`The data must be a number or a string value which is fetched from the database and projected up into the right corner.`

Are you sure in `must` here? I think it `could` be absolutely `any` value from `anywhere`, not just fetched from database.